index.html

```diff
@@ -118,6 +118,20 @@
       </pre>
       </div>
     </div>
+    
+    <div id="7-Technical-Sins" class="step slide" data-x="3000" data-y="0">
+      <q><b>Technical Sins :</b></q> <sup>[4a]</sup>
+      <br/>
+      <div style="font-size:20px;margin-top:5px;line-height:45px">
+        <li>Bugs and Potential Bugs</li>
+        <li>Coding Standards Breach</li>
+        <li>Duplications</li>
+        <li>Lack of Unit Tests</li>
+        <li>Bad Distribution of Complexity</li>
+        <li>Spaghetti Design</li>
+        <li>Not Enough or Too Many Comments</li>

```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Surround this <li> item tag by a <ul> or <ol> container one. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=Web%3AItemTagNotWithinContainerTagCheck)